### PR TITLE
Login page design

### PR DESCRIPTION
**Pull request by flymperopoulos**
Trent Design on login and main page done. Need to fix future effect on navbar.

### CODE REVIEW

**Comment by youralien** [comment]
@flymperopoulos can you explain the difference between
- sign up
- login with facebook
- login with email?

File: public/index.html [hunk 1]
```diff
@@ -29,7 +33,7 @@
                     <div class="collapse navbar-collapse">
                         <ul class="nav navbar-nav">
                             <li class="dropdown" ng-show = "checkLogin">
-                              <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Start<span class="caret"></span></a>
+                              <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" style="color:#3e454c">Start<span class="caret"></span></a>
                               <ul class="dropdown-menu" role="menu">
                                 <li><a ui-sref = "createQuestion" ng-show = "checkLogin">Create</a></li>
```
Comment (youralien) [inline public/index.html]: This create link is invisible, because it's white text on a very light grey background.

File: public/index.html [hunk 2]
```diff
@@ -12,12 +12,16 @@
...
     <link id="favicon" rel="icon" type="image/png" sizes="64x64" href="images/boolio-icon.png">
     <link href='stylesheets/main.css' rel='stylesheet'/>
+
+    <!-- font types -->
+    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Varela+Round" />
+
     <body ng-controller="mainController">
         <div ng-app = "myApp" >
-            <nav class="navbar navbar-default navbar-fixed-top">
+            <nav class="navbar navbar-default navbar-fixed-top" ng-show="navShow" style="background-color:#f3f3f4">
```
Comment (youralien) [inline public/index.html]: Not really sure how I feel about having styling within the html. Alternative: give these elements their own id's, and link the ids in the main.css file.

Comment (flymperopoulos) [inline public/index.html]: Yea I know, just did this to prototype it faster for tomorrow. Would refactor.

**Comment by youralien** [comment]
__Logo could be smaller, says Trent__

**Comment by flymperopoulos** [comment]
Yea, tried to make the transparent logo smaller and the quality and resolution decreases... we might want a new logo smaller pic

**Pull request merged**